docs/contents/pdf_model.rst

```diff
@@ -164,11 +167,51 @@ UML PDF model
             + [str] rel_path
             + [str] caption
             + [str] text: text inside Figure area
+            + [LTTextBox]: texts with positions
             --
             **id**: ""figure.<1,2,3,n>""
             **rel_path**: figures/<image>
         }
 
+        class HorizontalBox {
+            + [str] text
+            --
+            **text**: plain text in the box
+        }
+
+        class HorizontalLine {
+            + [str] text
+            --
+            **text**: plain text in the line
+        }
+
+        class Word {
+            + [str] text
```

Comment:
must this be an attribute? I think we can make this either a property or a function that concatenates all chars in the word